woocommerce/woocommerce-android
# [TTP] Revisit text and illustration on TTP flows m2

Closes: #8386


### Description

The PR:
* Changes the illustrations as per the design request
* Changes the TTP and Card Reader Icon on the "Select payment type" screen
* Adds a description to TTP and Card Reader row on the "Select payment type" screen

### Testing instructions

Run the TTP flow and notice:
* New illustrations
* New TTP and Card Reader Icons on the "Select payment type" screen
* New description to TTP and Card Reader row on the "Select payment type" screen

### Images/gif




https://user-images.githubusercontent.com/4923871/227137040-19248fee-f32b-48ff-94a1-d8d447e3bca8.mp4
